### DISCUSSION TITLE
Authentication Issues with Hoobs

### DISCUSSION THREAD

**Discussion by krakenbait** [homebridge-simplisafe3/homebridge-simplisafe3#248] [Q&A] [answered]
I'm not sure if it's a bug or not, but I am unable to get the plugin to work in Hoobs. Every time it fails to Authenticate. I understand that there must be some kind of pop-up according to the ReadMe file, but I can't seem to get it to work. The space underneath "Link Account" is a text field. I've been using the plugin for a few years now and this is the first time, I've had any issue. 

Any advice would be greatly appreciated. 

<img width="680" alt="Screen Shot 2021-10-14 at 6 44 32 PM" src="https://user-images.githubusercontent.com/80723669/137410067-af57c81e-436c-4a3b-85b2-d090ef07f351.png">

**Comment by shamoon** [comment, marked as answer]
Hi @krakenbait. Yea the new authentication process uses a part of Homebridge that may not be built into HOOBS. As a reminder, HOOBS is basically a for-profit entity that came from an older version of Homebridge, customers pay for hardware (I think?) and a system that actually just pulls open source software (like this one) made by people who aren't compensated for it. I don't have HOOBS, nor do we have plans to go out of our way to support it. In the past it has always just worked but it seems now this part doesn't , which is unfortunate.

All that said, the plug-in does include a command-line tool you can use instead, please see the README file for more info on that.

Hope that helps

**Reply by krakenbait** [reply]
Thanks for the input. I'll give Homebridge a shot. I'd much rather support open source anyways.

**Comment by mkellsy** [comment]
HOOBS is [open source](https://github.com/hoobs-org) too. We only sell Raspberry Pi kits.
HOOBS is actually the continuation of the original [homebridge-config-ui](https://www.npmjs.com/package/homebridge-config-ui) plugin.

HOOBS now has the Link Account button on the desktop app.

**Reply by Githubnewbynewb** [reply]
Appreciate the effort @mkellsy, see my post here about the blank screen after getting the Link Account button: https://github.com/hoobs-org/HOOBS/issues/1614

**Reply by shamoon** [reply]
> HOOBS is [open source](https://github.com/hoobs-org) too. 

HOOBS being open source doesn’t really have much to do with anything, does it…

**Reply by mkellsy** [reply]
Context
> HOOBS is open source too. **We only sell Raspberry Pi kits**.

**Comment by chuckbradley** [comment]
I'm not sure, but I think the answer is the desktop version of HOOBS app. When you use it, you get the OAuth sign on to the SimpliSafe app. Once that is done you don't need to worry about the credentials in the config.json anymore.
